projects/go-lib/src/lib/components/go-table/go-table.component.ts

```diff
@@ -490,8 +496,7 @@ export class GoTableComponent implements OnInit, OnChanges, AfterViewInit, OnDes
   }
 
   private setupRowSelectFormForCurrentPage(): void {
-    this.rowControlSubsCollection.unsubscribe();
-    this.rowControlSubsCollection = new Subscription();
+    this.pageChange$.next();
```

Comment:
Does calling `this.pageChange$.next();` here actually cleans up the subscription? I believe we would still either need to call `.complete()` or `.unsubscribe()` here and then set `this.pageChange$ = new Subject()` in order to actually clean up this subscription

Reply:
I think we're good here because of this part:

```javascript
  private setupRowSelectFormControlsSubs(): void {
    this.getDisplayData().forEach((row: any) => {
      this.rowSelectForm.get(`selection_${row[this.localTableConfig.selectBy]}`)
        .valueChanges.pipe(takeUntil(this.pageChange$))
        .subscribe((value: boolean) => {
          this.selectionChange(value, row);
        });
    });
  }
```

Here we're subscribing to each control, but then each of those subscriptions is watching `this.pageChanges$` and unsubscribing from the control when `this.pageChanges$` emits. 

Here's what happens if I log `pageChange$` every time the page changes and then switch pages a couple times:

![Screen Shot 2020-11-04 at 3 55 28 PM](https://user-images.githubusercontent.com/23343417/98167004-5d1d1400-1eb6-11eb-9b74-c91b191a899d.png)

If it was duplicating the subscriptions, I think you'd expect there to be 10, then 20, and then 20 `observers`.

So it's like the `pageChanges$` gets "emptied out" every time the page changes due to the `takeUntil`s and then is ready to receive a new set of observers. Does that make sense or am I missing something?